### PR TITLE
Feature/204 example jobs

### LINKED ISSUE #326: Possible stack overflow in JobGraphNodeBuilder.isEdgeShortcutFor(...)
I honestly don't know what the background of this is, but seems there's a recursive call here without a proper escape condition.

Happens in this job:

image

Whenever I interact with the Number Analyzer ...

```
Message:
    java.lang.StackOverflowError
Level:
    SEVERE
Stack Trace:
null
    edu.uci.ics.jung.graph.DirectedSparseGraph.containsVertex(DirectedSparseGraph.java:205)
    edu.uci.ics.jung.graph.DirectedSparseGraph.getInEdges(DirectedSparseGraph.java:132)
    org.datacleaner.widgets.visualization.JobGraphNodeBuilder.isEdgeShortcutFor(JobGraphNodeBuilder.java:164)
    org.datacleaner.widgets.visualization.JobGraphNodeBuilder.isEdgeShortcutFor(JobGraphNodeBuilder.java:172)
    org.datacleaner.widgets.visualization.JobGraphNodeBuilder.isEdgeShortcutFor(JobGraphNodeBuilder.java:172)
    org.datacleaner.widgets.visualization.JobGraphNodeBuilder.isEdgeShortcutFor(JobGraphNodeBuilder.java:172)
    org.datacleaner.widgets.visualization.JobGraphNodeBuilder.isEdgeShortcutFor(JobGraphNodeBuilder.java:172)
    org.datacleaner.widgets.visualization.JobGraphNodeBuilder.isEdgeShortcutFor(JobGraphNodeBuilder.java:172)
    org.datacleaner.widgets.visualization.JobGraphNodeBuilder.isEdgeShortcutFor(JobGraphNodeBuilder.java:172)
    org.datacleaner.widgets.visualization.JobGraphNodeBuilder.isEdgeShortcutFor(JobGraphNodeBuilder.java:172)
    org.datacleaner.widgets.visualization.JobGraphNodeBuilder.isEdgeShortcutFor(JobGraphNodeBuilder.java:172)
    org.datacleaner.widgets.visualization.JobGraphNodeBuilder.isEdgeShortcutFor(JobGraphNodeBuilder.java:172)
    org.datacleaner.widgets.visualization.JobGraphNodeBuilder.isEdgeShortcutFor(JobGraphNodeBuilder.java:172)
    org.datacleaner.widgets.visualization.JobGraphNodeBuilder.isEdgeShortcutFor(JobGraphNodeBuilder.java:172)
    org.datacleaner.widgets.visualization.JobGraphNodeBuilder.isEdgeShortcutFor(JobGraphNodeBuilder.java:172)
    org.datacleaner.widgets.visualization.JobGraphNodeBuilder.isEdgeShortcutFor(JobGraphNodeBuilder.java:172)
    org.datacleaner.widgets.visualization.JobGraphNodeBuilder.isEdgeShortcutFor(JobGraphNodeBuilder.java:172)
```

### PR DESCRIPTION
**Pull request by kaspersorensen**
Many improvements wrt... #204 Not sure it fixes it entirely, but a good step on the way :-)

Also fixes #326 and adds more fields to the example data file.

### CODE REVIEW

File: desktop/ui/src/main/resources/datacleaner-home/jobs/Address cleansing with EasyDQ.analysis.xml [hunk 1]
```diff
@@ -1,74 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<job xmlns="http://eobjects.org/analyzerbeans/job/1.0">
-    <job-metadata>
-        <job-description>Corrects and validates addresses (credentials reqd)</job-description>
```
Comment (tomaszguzialek) [inline desktop/ui/src/main/resources/datacleaner-home/jobs/Address cleansing with EasyDQ.analysis.xml]: Reqd - typo?

**Comment by tomaszguzialek** [comment]
Looks good to me!

File: desktop/ui/src/main/resources/datacleaner-home/jobs/Address cleansing with EasyDQ.analysis.xml (hunk 1 above)
Comment (kaspersorensen) [inline desktop/ui/src/main/resources/datacleaner-home/jobs/Address cleansing with EasyDQ.analysis.xml]: This part is deleted, so doesn't matter ... but "reqd" is a pretty common (I think) shorthand for "required"

Comment (tomaszguzialek) [inline desktop/ui/src/main/resources/datacleaner-home/jobs/Address cleansing with EasyDQ.analysis.xml]: Oh, true - it is deleted :). Anyways, I will merge it.

**Pull request merged**